<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QProgramTV</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Desktop/cinema-slate_318-1957.jpg</normaloff>../../Desktop/cinema-slate_318-1957.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="icon">
        <iconset>
         <normaloff>../build-QProgramTV-Desktop_Qt_5_8_0_MSVC2015_32bit-Release/release/img/singleChannel.png</normaloff>../build-QProgramTV-Desktop_Qt_5_8_0_MSVC2015_32bit-Release/release/img/singleChannel.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Spis filmów</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>220</height>
           </size>
          </property>
          <property name="title">
           <string>Wybierz gatunek i kanał</string>
          </property>
          <widget class="QComboBox" name="comboBox_Genre_Main">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>250</width>
             <height>20</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Gatunki filmów&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox_2">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>265</x>
             <y>20</y>
             <width>150</width>
             <height>20</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>5</x>
             <y>180</y>
             <width>170</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Wszystkich filmów:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>5</x>
             <y>160</y>
             <width>170</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QCheckBox" name="checkBox">
           <property name="geometry">
            <rect>
             <x>425</x>
             <y>20</y>
             <width>101</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Ulubione kanały</string>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox_Genre_1">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>250</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox_Genre_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>80</y>
             <width>250</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox_Genre_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>110</y>
             <width>250</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox_Genre_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>140</y>
             <width>250</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox_Genre_5">
           <property name="geometry">
            <rect>
             <x>265</x>
             <y>50</y>
             <width>250</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox_Genre_6">
           <property name="geometry">
            <rect>
             <x>265</x>
             <y>80</y>
             <width>250</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox_Genre_7">
           <property name="geometry">
            <rect>
             <x>265</x>
             <y>110</y>
             <width>250</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox_Genre_8">
           <property name="geometry">
            <rect>
             <x>265</x>
             <y>140</y>
             <width>250</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QTextBrowser" name="textBrowser">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="html">
           <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="icon">
        <iconset>
         <normaloff>../build-QProgramTV-Desktop_Qt_5_8_0_MSVC2015_32bit-Release/release/img/multiChannel.png</normaloff>../build-QProgramTV-Desktop_Qt_5_8_0_MSVC2015_32bit-Release/release/img/multiChannel.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Ulubione</string>
       </attribute>
       <widget class="QListView" name="listView">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonRemove">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>30</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Usuń</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonAdd">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>60</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Dodaj</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonClear">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>90</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Wyczyść</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxRemained">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>120</y>
          <width>211</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonSaveFavourites">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>220</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Zapisz</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonLoadFavourites">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>220</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Wczytaj</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_numberOfFavLeftChannels">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>300</y>
          <width>360</width>
          <height>16</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWersja_1_0">
    <property name="title">
     <string>Wersja 2.0</string>
    </property>
   </widget>
   <addaction name="menuWersja_1_0"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionWersja_1_0">
   <property name="text">
    <string>Wersja 1.0</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
